Outlines/simple-tcr.md

```diff
@@ -0,0 +1,18 @@
+# Proposed title
+Simple Token-Curated Registry
+
+## Structure
+1. What is Token-Curated Registry
+2. Examples of TCRs
+3. Creating a TCR
+	1. Smart contract
+	2. Front-end
+4. Limitations and improvement ideas
+5. Wrap-up
+
+## Motivation
+One of the biggest powers of Ethereum is the ability to easily program a system with a desired behaviour of its users. Different patterns of such mechanisms were discovered and then implemented. Token-curated registry is one such mechanism. In short, it allows to create a network of incentivised curators to create a list based on some criteria. Such a flexible tool opens a lot of possibilities for aspiring dapp developers.
+
+## Who should read this
+Anyone who is interested in creating decentralized applications as well as those who want to learn more about game theory and mechanism design.
+The article will include code samples in Solidity and JS and will require a moderate technical background.
```

Comment:
Good! Let's keep the Blockchain hype low and the practical how to build it high :)